cmd/slowlog-faker/main.go

```diff
@@ -0,0 +1,543 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/kshvakov/clickhouse"
+	"github.com/percona/go-mysql/event"
+	slowlog "github.com/percona/go-mysql/log"
+	parser "github.com/percona/go-mysql/log/slow"
+	"github.com/percona/go-mysql/query"
+)
+
+var opt = slowlog.Options{}
+
+const agentUUID = "dc889ca7be92a66f0a00f616f69ffa7b"
+
+func main() {
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	// zipf9 := rand.NewZipf(r, 5, 4, 9)
+	// zipf99 := rand.NewZipf(r, 5, 42, 99)
+
+	slowLogPath := flag.String("slowLogPath", "logs/mysql-slow.log", "Path to MySQL slow log file")
+	logTimeStart := flag.String("logTimeStart", "2019-01-01 00:00:00", "Start fake time of query from")
+	repeatN := flag.Int("repeatN", 0, "Scan slowlog given times (when 0 will wait for new evens)")
+	dsn := flag.String("dsn", "clickhouse://127.0.0.1:9000?database=pmm", "DSN of ClickHouse Server")
+
+	flag.Parse()
+	log.SetOutput(os.Stderr)
+
+	connect, err := sql.Open("clickhouse", *dsn)
+	if err != nil {
+		log.Fatal(err)
+	}
+	if err = connect.Ping(); err != nil {
+		if exception, ok := err.(*clickhouse.Exception); ok {
+			fmt.Printf("[%d] %s \n%s\n", exception.Code, exception.Message, exception.StackTrace)
+		} else {
+			fmt.Println(err)
+		}
+		return
+	}
+
+	events := parseSlowLog(*slowLogPath, opt)
+	fmt.Println("Parsing slowlog: ", *slowLogPath, "...")
+	logStart, _ := time.Parse("2006-01-02 15:04:05", *logTimeStart)
+	periodNumber := 0
+	iteration := 1
+	fmt.Printf("Iteration %d of %d \n", iteration, *repeatN)
+	for {
+		i := 0
+		aggregator := event.NewAggregator(true, 0, 0)
+		prewTs := time.Time{}
+		periodStart := time.Time{}
```

Comment:
```suggestion
		var periodStart time.Time
```

Do not use struct initialization syntax if you don't initialize anything; especially if you _can't_ initialize anything – `time.Time` has no public fields.